set.mm

```diff
@@ -528868,6 +528934,404 @@ Set induction (or epsilon induction)
       VNAVMVACVBVGUJABVGUEUFUGUHUIVEVKUKULABVGIVPRUMUNDVFUOUPUQABIURUS $.
   $}
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Transitive closure of a class
```

Comment:
Probably "Transitive closure of a relation" would be better, even if the definition is given for an arbitrary class, since the "transitive closure" of a class or set, when not in the context of binary relations, has a different meaning (even if related, since it is the class of predecessors under the transitive closure of the membership relation).

Reply:
Done